src/components/playback/playbackmanager.js

```diff
@@ -1316,16 +1317,20 @@ class PlaybackManager {
 
             return profiles.filter(function (p) {
                 if (p.Type === 'Video') {
-                    if (!p.AudioCodec) {
-                        return true;
-                    }
+                    const containers = (p.Container || '').toLowerCase().split(',');
```

Comment:
How safe is it to test strings directly?
I am currently using arrays and assume that it is joined exactly by `,`.

Reply:
Inspired by https://github.com/jellyfin/jellyfin/blob/c795f17fa6ae9e4f68f23cb569a1129de8fd7635/MediaBrowser.Model/Dlna/ContainerProfile.cs#L19-L72